pmm-server.env

```diff
@@ -0,0 +1,10 @@
+# env file passed to the container
+# full list of environment variables:
+# https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/docker.html#environment-variables
+
+# keep updates disabled
+# do image replacement instead (update the tag and restart the service)
+DISABLE_UPDATES=1
+
+# Enable DBaaS feature
+#ENABLE_DBAAS=1
```

Comment:
Why do we split environment variables between this file and service?

Reply:
service has environment for systemd/podman and env file is passed to pmm.

It is not split, it just another file to pass to env to pmm